pandas/core/arrays/numpy_.py

```diff
@@ -351,6 +352,27 @@ def interpolate(
             return self
         return type(self)._simple_new(out_data, dtype=self.dtype)
 
+    def take(
+        self,
+        indices: TakeIndexer,
+        *,
+        allow_fill: bool = False,
+        fill_value: Any = None,
+        axis: AxisInt = 0,
+    ) -> Self:
+        """
+        Take entries from this array at each index in a list of indices,
+        producing an array containing only those entries.
+        """
+        result = super().take(
+            indices, allow_fill=allow_fill, fill_value=fill_value, axis=axis
+        )
+        # See GH#62448.
+        if self.dtype.kind in "iub":
+            return type(self)(result, copy=False)
```

Comment:
i think this should use `result._ndarray` instead of `result`?  it doesn't actually make a difference, but feels wonky to be passing a mal-formed object here.